pkg/apis/components/v1alpha1/types.go

```diff
@@ -6,6 +6,9 @@
 package v1alpha1
 
 import (
+	"strconv"
+
+	"k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
```

Comment:
Why do we need to import v1beta1? seems like v1 has JSON struct as well.

Reply:
Changed to `v1`.